docs/getting-started/getting-started-rest/03-handling-errors.md

```diff
@@ -16,7 +16,6 @@ Using error models instead of raw status codes offers several advantages:
 2. **Clarity**: Error models provide clear, structured information about the error, including error codes, messages, and additional details. This helps developers understand what went wrong and how to fix it.
 3. **Extensibility**: Error models can be extended to include additional information, such as error details, validation issues, or links to documentation. This makes it easier to provide comprehensive error information.
 4. **Documentation**: Error models improve the generated API documentation by clearly defining the structure of error responses. This helps API consumers understand the possible error responses and how to handle them.
-5. **Type Safety**: In strongly-typed languages, using error models can provide type safety, ensuring that error responses conform to the expected structure.
```

Comment:
I feel this paragraph is still relevant though. Reason for removal?

Reply:
I took it out because I wasn't 100% sure it applied to TypeSpec, but if it does I can undo the removal.

Reply:
Added it back.